.ebextensions/kibana_proxy.config

```diff
@@ -0,0 +1,17 @@
+packages:
+  yum:
+    mod24_ssl: []
+
+files:
+  "/etc/httpd/conf.d/kibana.conf":
+    content: |
+      <VirtualHost *:80>
+        ServerName        kibana.safecast.cc
+
+        RequestHeader set Authorization: "Basic %{KIBANA_BASIC_AUTH}e" env=KIBANA_BASIC_AUTH
+
+        SSLProxyEngine on
+
+        ProxyPass         "/" "https://396401c55bfa44c6a493f1c4ee48774d.us-west-2.aws.found.io:9243/"
```

Comment:
Thought about making this an env var but the performance warning on https://httpd.apache.org/docs/2.4/mod/mod_proxy.html#proxypassinterpolateenv made me think we should just leave it in the code